roboteam_ai/src/control/shotControllers/ShotController.cpp

```diff
@@ -51,8 +51,7 @@ RobotCommand ShotController::getRobotCommand(world::Robot robot, const Vector2 &
     bool validAngle = robotAngleIsGood(robot, lineToDriveOver, precision);
 
     RobotCommand shotData;
-    // std::cout<<" Online: "<<isOnLineToBall <<" behind: "<<isBehindBall<<" valid: "<<validAngle<<" isShooting: " <<isShooting<<std::endl;
-    if (isOnLineToBall && isBehindBall && (validAngle || isShooting)) {
+    if (isBehindBall && ((isOnLineToBall && validAngle) || isShooting)) {
```

Comment:
So is this done so that it cancels out of not shooting less often, I think?

Reply:
Yes